library/c/api_store.go

```diff
@@ -4,30 +4,33 @@ package main
 #include <cgo_utils.h>
 */
 import "C"
-import "github.com/waku-org/go-waku/library"
+import (
+	"unsafe"
+
+	"github.com/waku-org/go-waku/library"
+)
 
 // Query historic messages using waku store protocol.
 // queryJSON must contain a valid json string with the following format:
 //
 //	{
 //		"pubsubTopic": "...", // optional string
```

Comment:
For store query either a pubsubTopic or a contentTopic should be present right. 
Maybe we should mention that here.

Reply:
It is possible for both fields to be empty to return all results with no filtering

Reply:
Probably. 
But we may want to keep the store protocol similar in the sense that wildcard store query like this should still be supported.
It would be interesting because how will the store client know if the storeNode is supporting all contentTopics it is interested in.
cc @SionoiS , store protocol may need an update to mandate contentTopic and/or pubsubTopic.